redaxo_loader.php

```diff
@@ -1,8 +1,52 @@
 <?php
+
 /**
- * Download latest REDAXO release from github
+ * Download REDAXO releases from github
  * License: MIT License
  */
+error_reporting(E_ALL);
+ini_set("display_errors", 1);
+
+// prevent Downloads from not intended URLs
+if(isset($_GET['url']) && isset($_GET['download']))
```

Comment:
ich würde die url in ´curl_file_get_contents` prüfen statt hier oben so out-of-context